src/components/IssueSearchAdvanced.vue

```diff
@@ -95,14 +100,37 @@ const optionsFor = (field: string) =>
     major_incident_state: flawIncidentStates,
     affects__affectedness: affectAffectedness,
   })[field] || null;
+
 const shouldShowAdvanced = ref(true);
+
+watch(queryFilterVisible, () => {
+  if (!queryFilterVisible.value) {
+    query.value = '';
+  }
+});
 </script>
 
 <template>
-  <details :open="shouldShowAdvanced" class="osim-advanced-search-container">
-    <summary class="mb-2" @click="shouldShowAdvanced = true">Advanced Search</summary>
+  <details :open="shouldShowAdvanced" class="osim-advanced-search-container container-fluid">
+    <summary class="mb-1" @click="shouldShowAdvanced = true">Advanced Search</summary>
     <form class="mb-2" @submit.prevent="submitAdvancedSearch">
-      <div v-for="(facet, index) in facets" :key="facet.field" class="input-group mb-2">
+      <div v-if="queryFilterVisible" class="input-group my-1">
+        <div type="button" class="form-control bg-secondary text-white pe-none py-0 d-flex" style="max-width: 13.125%;">
```

Comment:
Is the issue with the dynamic width we saw during the demo fixed?
The width of the `Query Filter` label changes with the window

Reply:
I fixed in other advance search PR, but probably makes sense to add it here.
(Will do tomorrow)